projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -7,6 +7,7 @@
 
 namespace Automattic\Jetpack\Forms\ContactForm;
 
+use Automattic\Jetpack\Assets;
```

Comment:
Since we're introducing the Assets package, we'll need to require it in the package's `composer.json` too.

Reply:
I think it's already there on trunk: https://github.com/Automattic/jetpack/blob/trunk/projects/packages/forms/composer.json#L72

Reply:
ah sorry, I had missed it!

I will say that having the requires at the bottom of the file and the require-dev at the top tricked me more than once. :) Maybe we could move those to the top, like we do for the other packages?

Reply:
Sure! Just moved it to the top 🙂